packages/oasis-actions/src/strategies/aave/close.ts

```diff
@@ -45,12 +45,17 @@ export async function close(
   const FEE_BASE = 10000
   const collectFeeFrom = args.collectSwapFeeFrom ?? 'sourceToken'
   const swapAmountBeforeFees = args.collateralAmountLockedInProtocolInWei
-  const fee = calculateFee(swapAmountBeforeFees, FEE, FEE_BASE)
 
-  const swapAmountAfterFees = swapAmountBeforeFees.minus(
-    collectFeeFrom === 'sourceToken' ? fee : ZERO,
+  const fee = Position.calculateFee(
```

Comment:
I think the older way was better.   Also the off by wei was in adjust i think so not quire sure if this fixes the issue